tests/functional/firefox/test_iostestflight.py

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
So it should be named `/functional/firefox/test_ios_testflight.py`?

The page is currently named `/pages/firefox/iostestflight.py` (which is why I named the test page `test_iostestflight.py`). Should the page file name be changed as well to `/pages/firefox/ios_testflight.py`?